release/models/system/openconfig-system.yang

```diff
@@ -46,7 +47,13 @@ module openconfig-system {
     Section 4.c of the IETF Trust's Legal Provisions Relating
     to IETF Documents (http://trustee.ietf.org/license-info).";
 
-  oc-ext:openconfig-version "0.16.1";
+  oc-ext:openconfig-version "0.16.2";
+  
+  revision "2022-12-20" {
+    description
+      "Added network instance leaf to ntp client.";
+    reference "0.16.2";
```

Comment:
New leaf additions have historically incremented the minor version - patch versions mostly just for doc changes/clarifications that are not adding/removing nodes to the tree

Reply:
I see Leo has updated the minor version